### PR TITLE
Initiale release of Instance isle bios

### PR DESCRIPTION
**Pull request by Atemo**
* **Addressed Issue(s)**: none

* **Server Mode**: renewal

* **Description of Pull Request**: Initiale release of Instance isle bios

### CODE REVIEW

**Comment by ecdarreola** [comment]
Works fine! 👍 Thanks!

**Comment by zackdreaver** [comment]
Is it Reaper Yanku(this PR), Reaper Yankou(iRO) or Reaper Ankou(npcidentity)?

**Comment by zackdreaver** [comment]
Test report:
Ankou spawned twice on the last area. Is it normal? haven't tested it on iRO.

**Comment by Atemo** [comment]
>Is it Reaper Yanku(this PR), Reaper Yankou(iRO) or Reaper Ankou(npcidentity)?

I used iRO text but it should be **Grim Reaper Ankou**

>Ankou spawned twice on the last area. Is it normal? haven't tested it on iRO.

The npc Ankou is enabled 2 times but there is only one spawn of Grim Reaper Ankou. Do you have 2 bosses? I don't find the bug. Yours steps?

**Comment by zackdreaver** [comment]
Let's see,
after beated the last spawn of horde,
I moved to top right,before the ankou spawn, then moved to the center area,
then there are 2 ankou spawned at the same time.

File: src/map/battle.cpp [hunk 1]
```diff
@@ -6226,6 +6226,11 @@ struct Damage battle_calc_magic_attack(struct block_list *src,struct block_list
 					case NPC_VENOMFOG:
 						skillratio += 600 + 100 * skill_lv;
 						break;
+					case NPC_COMET:
+						i = (sc ? distance_xy(target->x, target->y, sc->comet_x, sc->comet_y) : 8) / 2;
+						i = cap_value(i, 1, 4);
+						skillratio = 2500 + ((skill_lv) * 500);
```
Comment (secretdataz) [inline src/map/battle.cpp]: skill_lv - i

**Comment by slyx88** [comment]
Nice!
This NPC Comet seems suitable for Fenrir in MGT instance as well?

**Comment by Questune09** [comment]
Heroic coin exchanger should be included?

**Comment by Atemo** [comment]
>Heroic coin exchanger should be included?

I won't add it in this commit.

@zackdreaver I still can't reproduce the double boss spawn. Please check if the bug is coming from your side.

**Comment by Akaineko-astasi** [comment]
There's probably a typo in item_db.txt:
`15095,Hero_Judgement_Shawl,Hero Judgement Shawl,4,10,,500,,77,,1,0x00008110,63,2,16,,160,1,,{ bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus bMatk,10; bonus bBaseAtk,10; bonus bHit,10; bonus bFlee,10; bonus2 bSkillAtk,"AB_JUDEX",10; bonus2 bSkillAtk,"AB_ADORAMUS",10; bonus2 bSkillAtk,"SR_DRAGONCOMBO",10; bonus2 bSkillAtk,"SR_SKYNETBLOW",10; bonus2 bSkillAtk,"SR_EARTHSHAKER",10; .@r = getrefine(); bonus5 bAutoSpell,"PR_KYRIE",1,50+.@r,BF_SHORT,0; bonus5 bAutoSpell,"PR_LEXAETERNA",10,50+.@r,BF_MAGIC,1; },{},{}` 
shouldn't it be 
`15095,Hero_Judgement_Shawl,Hero Judgement Shawl,4,10,,500,,77,,1,0x00008110,63,2,16,,160,1,,{ bonus bMaxHPrate,5; bonus bMaxSPrate,5; bonus bMatk,10; bonus bBaseAtk,10; bonus bHit,10; bonus bFlee,10; bonus2 bSkillAtk,"AB_JUDEX",10; bonus2 bSkillAtk,"AB_ADORAMUS",10; bonus2 bSkillAtk,"SR_DRAGONCOMBO",10; bonus2 bSkillAtk,"SR_SKYNETBLOW",10; bonus2 bSkillAtk,"SR_EARTHSHAKER",10; .@r = getrefine(); bonus5 bAutoSpell,"PR_KYRIE",10,50+.@r,BF_SHORT,0; bonus5 bAutoSpell,"PR_LEXAETERNA",1,50+.@r,BF_MAGIC,1; },{},{}` 
(Kyrie lvl 10 and lex aeterna lvl 1 instead of vice versa);
and
`15098,Hero_Taget_Suits,Hero Target Suit,4,10,,200,,60,,1,0x00080808,56,2,16,,160,1,,{ bonus bMaxHPrate,5; bonus bMaxSPrate,5; .@r = getrefine(); bonus bAllStats,1; bonus5 bAutoSpell,"AC_CONCENTRATION",10,(5+.@r)*5,BG_WEAPON,0; bonus5 bAutoSpell,"AC_DOUBLE",10,(5+.@r)*5,BG_WEAPON,1; },{},{}` 
should be BF_WEAPON instead of BG_WEAPON: 
`15098,Hero_Taget_Suits,Hero Target Suit,4,10,,200,,60,,1,0x00080808,56,2,16,,160,1,,{ bonus bMaxHPrate,5; bonus bMaxSPrate,5; .@r = getrefine(); bonus bAllStats,1; bonus5 bAutoSpell,"AC_CONCENTRATION",10,(5+.@r)*5,BF_WEAPON,0; bonus5 bAutoSpell,"AC_DOUBLE",10,(5+.@r)*5,BF_WEAPON,1; },{},{}`

File: src/map/battle.cpp (hunk 1 above)
Comment (kukuasir1) [reply src/map/battle.cpp]: twRO
aegis:
```
SetSkill NPC_COMET //커미트-워록
OnInit:
	SkillMaxLv		5
	SkillProperty	PROPERTY_Nothing  // 속성없음
	SkillType		TOGROUND
	SkillPattern	SKILLPATT_ATTACKSPELL //마법공격
	SkillFlag		SKFLAG_APPLY_PLUSRNG //스킬사정거리증가옵션
	SkillRange                  11	    11	    11	    11	    11 // 스킬사정거리	
	SkillData1	  3000	  3500	  4000	  4500	  5000 // 범위1의 MATK 공격증가률
	SkillData2	  2500    3000    3500    4000    4500 // 범위2의 MATK 공격증가률
	SkillData3	  2000    2500    3000    3500    4000 // 범위3의 MATK 공격증가률
	SkillData4	  1500    2000    2500    3000    3500 // 범위4의 MATK 공격증가률
return
```

skillratio = 2500 + ((skill_lv - i + 1) * 500);

**Pull request merged**